src/ext/java/org/opentripplanner/ext/siri/SiriTimetableSnapshotSource.java

```diff
@@ -912,7 +913,7 @@ private boolean handleModifiedTrip(
           lineRef,
           vehicleRef
         );
-        return false;
+        return List.of(new UpdateError(null, UNKNOWN));
```

Comment:
`TRIP_ID_NOT_FOUND` Or a more specific error about the fuzzy trip matching failing, which could also be used for GTFS-RT